adafruit_featherwing/gps_featherwing.py

```diff
@@ -0,0 +1,183 @@
+# The MIT License (MIT)
+#
+# Copyright (c) 2019 Melissa LeBlanc-Williams for Adafruit Industries LLC
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+"""
+`adafruit_featherwing.gps_featherwing`
+====================================================
+
+Helper for using the `Ultimate GPS FeatherWing <https://www.adafruit.com/product/3133>`_.
+
+* Author(s): Melissa LeBlanc-Williams
+"""
+
+__version__ = "0.0.0-auto.0"
+__repo__ = "https://github.com/adafruit/Adafruit_CircuitPython_FeatherWing.git"
+
+import busio
+import adafruit_gps
+from adafruit_featherwing import shared
+
+class GPSFeatherWing:
+    """Class representing an `Ultimate GPS FeatherWing
+       <https://www.adafruit.com/product/3133>`_.
+
+       Automatically uses the feather's I2C bus."""
+    def __init__(self, update_period=1000, baudrate=9600):
+        """
+        :param int update_period: (Optional) The amount of time in milliseconds between
+                                  updates (default=1000)
+        :param int baudrate: (Optional) The Serial Connection speed to the GPS (default=9600)
+        """
+        if not isinstance(update_period, int):
+            raise ValueError("Update Frequency should be an integer in milliseconds")
+        if update_period < 250:
+            raise ValueError("Update Frequency be at least 250 milliseconds")
+        timeout = update_period // 1000 + 2
+        if timeout < 3:
+            timeout = 3
+
+        self._uart = busio.UART(shared.TX, shared.RX, baudrate=baudrate, timeout=timeout)
+        self._gps = adafruit_gps.GPS(self._uart, debug=False)
+        # Turn on the basic GGA and RMC info
+        self._gps.send_command(b'PMTK314,0,1,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0')
+        self._gps.send_command(b'PMTK220,{}'.format(update_period))
+
+    def update(self):
+        """
+        Make sure to call gps.update() every loop iteration and at least twice
```

Comment:
If you wan it to show up as code in the docs, you can put double backticks on either side (``). Single backticks will cause sphinx to attempt to do something with it, double tells sphinx that you're trying to display something in fixed-width font.

Reply:
I'm referring to `gps.update()`.

Reply:
Thanks! I didn't know that.